### PR TITLE
Angela/logs get started content update

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: products/logs/src/content/get-started/enable-destinations/aws-s3/index.md [hunk 1]
```diff
@@ -49,20 +49,20 @@ Once connected, Cloudflare lists Amazon S3 as a connected service under **Logs**
...
 
 <EnableReadPermissions/>
 
-For illustrative purposes, imagine that you want to store logs in the bucket *burritobot*, in the *logs* directory. The S3 URL would then be `s3://burritobot/logs`.
+For illustrative purposes, imagine that you want to store logs in the bucket `burritobot`, in the `logs` directory. The S3 URL would then be `s3://burritobot/logs`.
 
 To enable Logpush to Amazon S3:
 
-1. Create an S3 bucket. *See [instructions from Amazon](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html)*. Note: buckets in China regions (cn-north-1, cn-northwest-1) are currently not supported.
+1. Create an S3 bucket. Refer to [instructions from Amazon](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html). Note: buckets in China regions (cn-north-1, cn-northwest-1) are currently not supported.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/aws-s3/index.md]:
```suggestion
1. Create an S3 bucket. Refer to [instructions from Amazon](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).

    Note: Buckets in China regions (`cn-north-1`, `cn-northwest-1`) are currently not supported.
```

File: products/logs/src/content/get-started/enable-destinations/aws-s3/index.md [hunk 2]
```diff
@@ -49,20 +49,20 @@ Once connected, Cloudflare lists Amazon S3 as a connected service under **Logs**
...
 
-1. Edit and paste the policy below into **S3** > **Bucket** > **Permissions** > **Bucket Policy** (make sure to replace the *Resource* value with your own bucket path):
+1. Edit and paste the policy below into **S3** > **Bucket** > **Permissions** > **Bucket Policy** (make sure to replace the `Resource` value with your own bucket path):
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/aws-s3/index.md]:
```suggestion
1. Edit and paste the policy below into **S3** > **Bucket** > **Permissions** > **Bucket Policy**, replacing the `Resource` value with your own bucket path:
```

File: products/logs/src/content/get-started/enable-destinations/aws-s3/index.md [hunk 3]
```diff
@@ -88,5 +88,5 @@ To enable Logpush to Amazon S3:
 
 <Aside type="note" header="Note">
 
-Logpush uses multipart upload for S3. Aborted uploads will result in incomplete files remaining in your bucket. To minimize your storage costs, Amazon recommends configuring a lifecycle rule using the `AbortIncompleteMultipartUpload` action. *See [Uploading and copying objects using multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config)*.
+Logpush uses multipart upload for S3. Aborted uploads will result in incomplete files remaining in your bucket. To minimize your storage costs, Amazon recommends configuring a lifecycle rule using the `AbortIncompleteMultipartUpload` action. Refer to [Uploading and copying objects using multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config).
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/aws-s3/index.md:93]:
```suggestion
Logpush uses multipart upload for S3. Aborted uploads will result in incomplete files remaining in your bucket. To minimize your storage costs, Amazon recommends configuring a lifecycle rule using the `AbortIncompleteMultipartUpload` action. Refer to [Uploading and copying objects using multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config).

```

File: products/logs/src/content/get-started/enable-destinations/azure/index.md [hunk 4]
```diff
@@ -45,15 +45,15 @@ Once connected, Cloudflare lists Microsoft Azure as a connected service under **
...
+1. Create a shared access signature (SAS). To learn about shared access signatures, refer to [information from Azure](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview). Logpush requires a service-level SAS, which provides the most restricted access. To create a service-level SAS, refer to [instructions from Azure](https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas) or use the **Storage Explorer** feature in your storage account portal. Select **Storage Explorer**, navigate to **Blob Containers** and then right-click on your blob container to see the **Get Shared Access Signature** option. Select that option, set an expiration time of at least 5 years, and select only **Write** permission.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/azure/index.md]: Nit: There are a lot of instructions in this paragraph... I understand that they may be optional, but consider alternatives like a child numbered list or a separate section.

File: products/logs/src/content/get-started/enable-destinations/bigquery/index.md [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
+pcx-content-type: how-to
 ---
 
-# Enable BigQuery
+# Enable Logpush to BigQuery
 
 Configure Logpush to send batches of Cloudflare logs to BigQuery.
 
-BigQuery supports loading up to 1,500 jobs per table per day (including failures) with up to 10 million files in each load. That means you can load into BigQuery once per minute and include up to 10 million files in a load. See BigQuery's quotas for load jobs for more information.  
+BigQuery supports loading up to 1,500 jobs per table per day (including failures) with up to 10 million files in each load. That means you can load into BigQuery once per minute and include up to 10 million files in a load For more information, refer to BigQuery's quotas for load jobs.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/bigquery/index.md]:
```suggestion
BigQuery supports loading up to 1,500 jobs per table per day (including failures) with up to 10 million files in each load. That means you can load into BigQuery once per minute and include up to 10 million files in a load. For more information, refer to BigQuery's quotas for load jobs.
```

File: products/logs/src/content/get-started/enable-destinations/azure/index.md [hunk 6]
```diff
@@ -45,15 +45,15 @@ Once connected, Cloudflare lists Microsoft Azure as a connected service under **
 
 ## Manage via API
 
-Cloudflare uses a service-level shared access signature (SAS) to gain access to your Blob Storage container. You'll need to provide *Write* permission and an expiration period of at least 5 years, which will allow you to not worry about the SAS token expiring.
+Cloudflare uses a service-level shared access signature (SAS) to gain access to your Blob Storage container. You will need to provide *Write* permission and an expiration period of at least 5 years, which will allow you to not worry about the SAS token expiring.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/azure/index.md]:
```suggestion
Cloudflare uses a service-level shared access signature (SAS) to gain access to your Blob Storage container. You will need to provide `Write` permission and an expiration period of at least 5 years, which will allow you to not worry about the SAS token expiring.
```

File: products/logs/src/content/get-started/enable-destinations/s3-compatible-endpoints/index.md [hunk 7]
```diff
@@ -80,11 +79,11 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
 ```
 <Aside type="note" header="Note">
 
-`<ENDPOINT_URL>` is the URL without the bucket name or path. Example: `endpoint=sfo2.digitaloceanspaces.com`
+`<ENDPOINT_URL>` is the URL without the bucket name or path. For example: `endpoint=sfo2.digitaloceanspaces.com`
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/s3-compatible-endpoints/index.md]:
```suggestion
`<ENDPOINT_URL>` is the URL without the bucket name or path. For example: `endpoint=sfo2.digitaloceanspaces.com`.
```

File: products/logs/src/content/get-started/enable-destinations/splunk/index.md [hunk 8]
```diff
@@ -64,29 +64,29 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
...
+  * `<SPLUNK_ENDPOINT_URL>`: The Splunk raw HTTP Event Collector URL with port. For example: `splunk.cf-analytics.com:8088/services/collector/raw`. 
+      * Cloudflare expects the HEC network port to be configured to `:443` or `:8088`. 
+      * Cloudflare expects the Splunk endpoint to be `/services/collector/raw` while configuring and setting up the Logpush job.  
+      * Ensure you have enabled HEC in Splunk. Refer to [Splunk Analytics Integrations](https://developers.cloudflare.com/fundamentals/data-products/analytics-integrations/splunk) for information on how to set up HEC in Splunk. 
   * `<SPLUNK_CHANNEL_ID>`: A unique channel ID. This is a random GUID that you can generate by:
-      * Using an online tool like the [GUID generator](https://www.guidgenerator.com/) 
-      * Using command line.  Example: `python -c 'import uuid; print(uuid.uuid4())'` 
+      * Using an online tool like the [GUID generator](https://www.guidgenerator.com/). 
+      * Using command line.  For example: `python -c 'import uuid; print(uuid.uuid4())'`.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/splunk/index.md]:
```suggestion
      * Using the command line.  For example: `python -c 'import uuid; print(uuid.uuid4())'`. 
```

File: products/logs/src/content/get-started/enable-destinations/splunk/index.md [hunk 9]
```diff
@@ -64,29 +64,29 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
...
+      * Ensure you have enabled HEC in Splunk. Refer to [Splunk Analytics Integrations](https://developers.cloudflare.com/fundamentals/data-products/analytics-integrations/splunk) for information on how to set up HEC in Splunk. 
   * `<SPLUNK_CHANNEL_ID>`: A unique channel ID. This is a random GUID that you can generate by:
-      * Using an online tool like the [GUID generator](https://www.guidgenerator.com/) 
-      * Using command line.  Example: `python -c 'import uuid; print(uuid.uuid4())'` 
+      * Using an online tool like the [GUID generator](https://www.guidgenerator.com/). 
+      * Using command line.  For example: `python -c 'import uuid; print(uuid.uuid4())'`. 
   * `<INSECURE_SKIP_VERIFY>`: Boolean value. Cloudflare recommends setting this value to `false`. Setting this value to `true` is equivalent to using the `-k` option with `curl` as shown in Splunk examples and is **not** recommended. Only set this value to `true` when HEC uses a self-signed certificate.
 
 <Aside type="note" header="Note">
 Cloudflare highly recommends setting this value to <code class="InlineCode">false</code>. Refer to the <a href="/faq#logpush-faq">Logpush FAQ</a> for more information.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/splunk/index.md:77]: Nit: If there is an empty line (it must really be empty, without even a single space) before and after the `<Aside ... >` and `</Aside>` tags you can use Markdown in the aside text, without having to use HTML.

File: products/logs/src/content/get-started/enable-destinations/splunk/index.md [hunk 10]
```diff
@@ -64,29 +64,29 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
...
 </Aside>
 
-  * `<SOURCE_TYPE>`: The Splunk sourcetype. Example: `cloudflare:json`.
-  * `<SPLUNK_AUTH_TOKEN>`: The Splunk authorization token that is URL-encoded. Example: `Splunk%20e6d94e8c-5792-4ad1-be3c-29bcaee0197d`.
+  * `<SOURCE_TYPE>`: The Splunk sourcetype. For example: `cloudflare:json`.
```
Comment (pedrosousa) [inline products/logs/src/content/get-started/enable-destinations/splunk/index.md]:
```suggestion
  * `<SOURCE_TYPE>`: The Splunk source type. For example: `cloudflare:json`.
```
Splunk uses `source type` (two words) in their documentation.

**Review by pedrosousa — APPROVED** [review]

File: products/logs/src/content/get-started/enable-destinations/splunk/index.md [hunk 11]
```diff
@@ -64,29 +64,29 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
...
+      * Using the command line.  For example: `python -c 'import uuid; print(uuid.uuid4())'`. 
   * `<INSECURE_SKIP_VERIFY>`: Boolean value. Cloudflare recommends setting this value to `false`. Setting this value to `true` is equivalent to using the `-k` option with `curl` as shown in Splunk examples and is **not** recommended. Only set this value to `true` when HEC uses a self-signed certificate.
 
 <Aside type="note" header="Note">
 Cloudflare highly recommends setting this value to <code class="InlineCode">false</code>. Refer to the <a href="/faq#logpush-faq">Logpush FAQ</a> for more information.
 </Aside>
 
-  * `<SOURCE_TYPE>`: The Splunk sourcetype. Example: `cloudflare:json`.
-  * `<SPLUNK_AUTH_TOKEN>`: The Splunk authorization token that is URL-encoded. Example: `Splunk%20e6d94e8c-5792-4ad1-be3c-29bcaee0197d`.
+  * `<SOURCE_TYPE>`: The Splunk source type. For example: `cloudflare:json`.
```
Comment (soheiokamoto) [inline products/logs/src/content/get-started/enable-destinations/splunk/index.md:80]: nit: Curious if this is all one word for all of them. The actual Splunk parameter is `sourcetype`.

Comment (pedrosousa) [reply products/logs/src/content/get-started/enable-destinations/splunk/index.md:80]: I've checked and they're not consistent across their documentation.

Examples:
https://docs.splunk.com/Documentation/Splunk/8.2.4/HadoopAnalytics/Addasourcetype (mixed occurrences)
https://docs.splunk.com/Documentation/Splunk/8.2.4/Data/Whysourcetypesmatter (two words, except for the parameter name)

Comment (angelampcosta) [reply products/logs/src/content/get-started/enable-destinations/splunk/index.md:80]: I think I would keep it as two words when we mention the parameter in the documentation, for instance "<SOURCE_TYPE>: The Splunk source type." But use the one word form for the examples, like in "destination_conf":"splunk://<SPLUNK_ENDPOINT_URL>?channel=<SPLUNK_CHANNEL_ID>&insecure-skip-verify=<INSECURE_SKIP_VERIFY>&sourcetype=<SOURCE_TYPE>&header_Authorization=<SPLUNK_AUTH_TOKEN>", "logpull_options": "fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID&timestamps=rfc3339", "dataset": "http_requests"}' | jq .

**Pull request merged**